src/services/railcontent.js

```diff
@@ -926,8 +914,8 @@ export async function fetchPlaylistItem(payload) {
   return await fetchHandler(url)
 }
 
-export async function postContentCompleted(contentId) {
-  let url = `/content/user/progress/complete`
+export async function postContentComplete(contentId) {
+  let url = `/content/v1/user/progress/complete`
   return postDataHandler(url, { contentId: contentId })
```

Comment:
Your BE changes include the contentId as part of the url route, not as part of the request body. I'm almost positive you'll need something like this:
```suggestion
  let url = `/content/v1/user/progress/complete/${contentId}`
  return postDataHandler(url)
```